web_src/js/markup/mermaid.js

```diff
@@ -8,8 +8,9 @@ function displayError(el, err) {
   el.closest('pre').before(errorNode);
 }
 
-export async function renderMermaid(els) {
-  if (!els || !els.length) return;
+export async function renderMermaid() {
+  const els = document.querySelectorAll('.markup code.language-mermaid');
+  if (!els?.length) return;
```

Comment:
`els` can not be `null`, https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll#return_value